### PR TITLE
Samples for Kotlin Cloud Functions

### PR DESCRIPTION
**Pull request by bshaffer**

### CODE REVIEW

**Comment by ace-n** [comment]
cc @grant @stew-r as FYI

**Comment by bshaffer** [comment]
@lesv added `shadowJar` step! I would love to remove gradle wrappers but I'd like to get @saturnism's take before doing so.

File: run/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+https://github.com/knative/docs/tree/master/docs/serving/samples/hello-world/helloworld-kotlin
```
Comment (lesv) [inline run/README.md]: Is this meant to be here?

Comment (bshaffer) [reply run/README.md]: oops. it sure isnt.

**Pull request merged**